extensions/sql-database-projects/src/test/testContext.ts

```diff
@@ -8,16 +8,11 @@ import * as azdata from 'azdata';
 import * as path from 'path';
 import * as TypeMoq from 'typemoq';
 import * as mssql from '../../../mssql/src/mssql';
-import * as vscodeMssql from 'vscode-mssql';
 
 export interface TestContext {
 	context: vscode.ExtensionContext;
 	dacFxService: TypeMoq.IMock<mssql.IDacFxService>;
-	azureFunctionService: TypeMoq.IMock<vscodeMssql.IAzureFunctionsService>;
 	outputChannel: vscode.OutputChannel;
-	vscodeMssqlIExtension: TypeMoq.IMock<vscodeMssql.IExtension>
-	dacFxMssqlService: TypeMoq.IMock<vscodeMssql.IDacFxService>;
-	schemaCompareService: TypeMoq.IMock<vscodeMssql.ISchemaCompareService>;
```

Comment:
Curious if our sqlproj friends (@kisantia, @Benjin) would like to keep any of these mocked services around for future use

Reply:
Yeah, feel free to let me know since I added this when creating tests for addSQLBindingQuickPick enhancements.